Amazon Alexa Reviews Analysis/Model/Amazon_Alexa_Reviews_Analysis.py

```diff
@@ -0,0 +1,101 @@
+# Amazon Alexa Reviews Analysis
+
+# The aim is to analyse Alexa's reviews by NLP. If the feedback is positive, the result is 1, else it is 0. Using logistic regression, I have tried to classify the feedback as positive or negative.
+
+# Importing Libraries
+
+import pandas as pd
+import nltk 
+nltk.download('stopwords')                 # download the stopwords from NLTK
+
+import string                              # for string operations
+
+from nltk.corpus import stopwords          # module for stop words that come with NLTK
+from nltk.stem import PorterStemmer        # module for stemming
+from nltk.tokenize import TweetTokenizer   # module for tokenizing strings
+
+from sklearn.linear_model import LogisticRegression  
+from sklearn.feature_extraction.text import CountVectorizer  
+from sklearn.model_selection import train_test_split  
+from sklearn.metrics import roc_auc_score
+
+# Getting our Data
+
+df = pd.read_csv(r'C:\Users\DELL\Desktop\Kaggle+HE\Github GSSoC21\Amazing Python Scripts\Amazon Alexa Reviews Analysis\Dataset\amazon_alexa.csv')
```

Comment:
use relative paths instead of your local system path. 
`./Amazon Alexa Reviews Analysis/dataset/amazon_alexa.csv`